ytsync/manager.go

```diff
@@ -149,7 +159,11 @@ func (s SyncManager) MarkVideoStatus(channelID string, videoID string, status st
 	res, _ := http.PostForm(endpoint, vals)
 	defer res.Body.Close()
 	body, _ := ioutil.ReadAll(res.Body)
-	var response apiSyncUpdateResponse
+	var response struct {
```

Comment:
why are we not using the aligned code in the api package? I would make sure we are making things DRY. `api.Response`